### ISSUE TITLE
Dynamic content announcement not working on grafics with dynamic alt text

### ISSUE THREAD

**Issue by Adriani90** [nvaccess/nvda#8495]
### Steps to reproduce:
1. Go to NVDA menu and make sure that announcement of dynamic content is enabled in object presentation
2. Open this website
http://www.uebersetzung-deutsch-italienisch.de/
3. Press h to go to the next heading and then press down arrow key once to land on the grafic
4. Wait some seconds and press upa and down arrow to check if the alt text of the grafic is the same
5. Use a braille display while the NVDA cursor is on the grafic

### Actual behavior:
NVDA shows the new alt text on the braille display right when it changes. But the speech handler does not fire any announcement by the synthesizer. to read the new alt text by using the synthesizer, the user must leave the grafic and return to it.

### Expected behavior:
Both braille and speech should announce the new alt text when it changes.

#### NVDA Installed/portable/running from source:
Both

#### NVDA version:
2018.2.1.

#### Windows version:
Windows 10

#### Name and version of other software in use when reproducing the issue:
Firefox, IE, Google Chrome (last updates)

**Comment by Adriani90** [comment]
The website is not working and I don't have any test case at hand. Closing as abandoned until I find another test case.
